pkg/internalapis/cnsoperator/config/triggercsifullsync_crd.yaml

```diff
@@ -9,66 +14,85 @@ spec:
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: Spec defines a specification of the TriggerCsiFullSync.
+            properties:
+              triggerSyncID:
+                description: TriggerSyncID gives an option to trigger full sync on
+                  demand. Initial value will be 0. In order to trigger a full sync,
+                  user has to set a number that is 1 greater than the previous one.
+                format: int64
+                type: integer
+            required:
+            - triggerSyncID
+            type: object
+          status:
+            description: Status represents the current information/status for the
+              TriggerCsiFullSync request.
+            properties:
+              error:
+                description: The last error encountered during CSI full sync operation,
+                  if any. Previous error will be cleared when a new full sync is in
+                  progress.
+                type: string
+              inProgress:
+                description: InProgress indicates whether a CSI full sync is in progress.
+                  If full sync is completed this field will be unset.
+                type: boolean
+              lastRunEndTimeStamp:
+                description: LastRunEndTimeStamp indicates last run full sync end
+                  timestamp. This timestamp can be either the successful or failed
+                  full sync end timestamp.
+                format: date-time
+                type: string
+              lastRunStartTimeStamp:
+                description: LastRunStartTimeStamp indicates last run full sync start
+                  timestamp. This timestamp can be either the successful or failed
+                  full sync start timestamp.
+                format: date-time
+                type: string
+              lastSuccessfulEndTimeStamp:
+                description: LastSuccessfulEndTimeStamp indicates last successful
+                  full sync end timestamp.
+                format: date-time
+                type: string
+              lastSuccessfulStartTimeStamp:
+                description: LastSuccessfulStartTimeStamp indicates last successful
+                  full sync start timestamp.
+                format: date-time
+                type: string
+              lastTriggerSyncID:
+                description: LastTriggerSyncID indicates the last trigger sync Id.
+                format: int64
+                type: integer
+            required:
+            - inProgress
+            - lastTriggerSyncID
+            type: object
+        type: object
     served: true
     storage: true
+    subresources:
```

Comment:
With status added as subResource, it is preventing operations like modify/update/add on status field. Please remove the subresources section